ttnn/cpp/ttnn/operations/eltwise/unary_backward/unary_backward.hpp

```diff
@@ -155,10 +155,9 @@ struct ExecuteUnaryBackwardOptionalFloat {
         const Tensor &input_tensor_arg,
         float parameter,
         const std::optional<MemoryConfig> &memory_config = std::nullopt,
-        const std::vector<bool> &are_required_outputs = std::vector<bool>{true},
         std::optional<Tensor> input_grad = std::nullopt) {
         auto output_memory_config = memory_config.value_or(input_tensor_arg.memory_config());
-        return OpHandler<unary_backward_op_type>::handle(queue_id, grad_tensor_arg, input_tensor_arg, parameter, output_memory_config, are_required_outputs, input_grad);
+        return OpHandler<unary_backward_op_type>::handle(queue_id, grad_tensor_arg, input_tensor_arg, parameter, output_memory_config, input_grad);
```

Comment:
is it possible to remove OpHandler usage for the ops in this PR and use ::invoke instead ?